templates/html-header.twig

```diff
@@ -8,6 +8,8 @@
 
 	{% include 'blocks/google_tag_manager.twig' %}
 
+	{% include 'blocks/hubspot_signups.twig' %}
```

Comment:
I also have [some code in my PR for HubSpot cookie consent](https://github.com/greenpeace/planet4-master-theme/pull/1390/files#diff-8479f762f16d76dd3afe830c872b63bab77ba39e3395d34d1123e65a08e6a7baR4-R12) that can be added to the head. Perhaps we can include a check in the twig template for whether the HubSpot plugin is active on the instance, and only output HS related things if needed?

Reply:
[I added a common `head/hubspot.twig` template](https://github.com/greenpeace/planet4-master-theme/pull/1390/files#diff-8479f762f16d76dd3afe830c872b63bab77ba39e3395d34d1123e65a08e6a7baR11-R14) where we can put both things, and the plugin check in my PR. Probably best to not add any more Twig templates to the `blocks` directory, since the name is misleading. Twig actually has something called a "block" (e.g. `{% block head %}{% endblock %}`), but what we include in that directory is not that, it's just other templates not wrapped in a block.

Reply:
Updated! 👍